pallets/gear/src/tests.rs

```diff
@@ -15086,6 +15086,80 @@ fn test_handle_signal_wait() {
     });
 }
 
+#[test]
+fn test_constructor_if_else() {
+    use demo_constructor::{Arg, Call, Calls, Scheme, WASM_BINARY};
+
+    init_logger();
+    new_test_ext().execute_with(|| {
+        let init = Calls::builder().bool("switch", false);
+        let handle = Calls::builder()
+            .if_else(
+                Arg::get("switch"),
+                Calls::builder().add_call(Call::Bool(false)),
```

Comment:
the pr introduces vec of calls, so add here vec of something.
for example, if true than panic(expected message), else wait .

Reply:
furthermore, as I said you can test in in examples/constructor crate exactly, you don't need wasm execution here

Reply:
The multiple calls will end up being tested anyways when I rewrite the signal test.